projects/storefrontapp-e2e-cypress/cypress/e2e/regression/quote/quote-configurator.e2e-flaky.cy.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as quote from '../../../helpers/quote';
+import * as common from '../../../helpers/common';
+import * as configurationCart from '../../../helpers/product-configurator-cart';
+import * as configuratorOverview from '../../../helpers/product-configurator-overview';
+
+const POWERTOOLS = 'powertools-spa';
+const testProductConfigurable = 'CONF_BANDSAW_ML';
+const testProductConfigurableWithIssues = 'CONF_SCREWDRIVER_S';
+const testProductConfigurableTextfield = '2116282';
+const EMAIL = 'gi.sun@pronto-hw.com';
+const PASSWORD = '12341234';
+const USER = 'Gi Sun';
+
+context('Quote<->Configurator integration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    quote.login(EMAIL, PASSWORD, USER);
+  });
+
+  describe('Request quote process with VC configurable product', () => {
+    it('should result in a quote in draft state if default configuration has no issues', () => {
+      quote.requestQuote(POWERTOOLS, testProductConfigurable, '1');
+      quote.checkQuoteIsDraft();
+    });
+
+    it('should not allow to request quote if configuration has issues', () => {
+      common.goToPDPage(POWERTOOLS, testProductConfigurableWithIssues);
+      quote.setQtyOnPD('1');
+      common.clickOnAddToCartBtnOnPD();
+      common.clickOnViewCartBtnOnPD();
+      quote.clickOnRequestQuoteInCart();
+
+      //we are still in cart, for now just check that
+      //TODO check for messages once https://jira.tools.sap/browse/CXSPA-4079 is done
+      cy.get('cx-cart-details').should('exist');
+      quote.clearCart();
```

Comment:
this fails when i execute the tests (and hence all subsequent tests, as the cart contains product with issues, hence no quote can be created)
![image](https://github.com/SAP/spartacus/assets/53090888/ae3b8a48-985c-4b5f-a4e4-942194c73823)
Additionally a more robust approach would be to move this to a beforeEach hook, that clears the cart if required.

Reply:
I replaced that with a better, simpler method from the cart e2e tools (but did not do that in beforeEach, as I struggled to get that stable. I didn't want to build the tools for that in the cart area, seemed to be over-engineered).

Reply:
@ChristophHi still not working for me, because it seems that the @delete_cart_item alias is registered for electronics store route, while the test runs in powertools.
![image](https://github.com/SAP/spartacus/assets/53090888/5fc712cf-c78f-4598-88b6-dc3da542c870)
![image](https://github.com/SAP/spartacus/assets/53090888/6d7699de-9610-4174-98f5-05f9b91408cf)
How are you running the tests (maybe i missed something?)
i just run `npm run cy:open` and selected your spec?

Reply:
ok found it myself - when starting cypres with `npm run cy:open:b2b` it works fine